plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -389,6 +389,7 @@ void GSSettingsDlg::UpdateControls()
 		ShowWindow(GetDlgItem(m_hWnd, IDC_ACCURATE_BLEND_UNIT), ogl ? SW_SHOW : SW_HIDE);
 		ShowWindow(GetDlgItem(m_hWnd, IDC_ACCURATE_BLEND_UNIT_TEXT), ogl ? SW_SHOW : SW_HIDE);
 		ShowWindow(GetDlgItem(m_hWnd, IDC_TC_DEPTH), ogl ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_UNSCALE_POINT_LINE), ogl ? SW_SHOW : SW_HIDE);
```

Comment:
Oops. I just followed the TC_DEPTH example. I guess the line above it can be removed as well, as the depth option has been moved to the hacks dialog.